contracts/CoveragePool.sol

```diff
@@ -51,19 +56,33 @@ contract CoveragePool is Ownable {
         collateralToken = _assetPool.collateralToken();
     }
 
+    /// @notice Approves the first risk manager
+    /// @dev Can be called only by the contract owner. Can be called only once.
+    ///      Does not require any further calls to any functions.
+    /// @param riskManager Risk manager that will be approved.
+    function approveFirstRiskManager(address riskManager) external onlyOwner {
+        require(
+            !firstRiskManagerApproved,
+            "The first risk manager is already approved"
```

Comment:
I'd say "The first risk manager was approved" because there is no guarantee it is still approved given that there is an unapproval process.